sentry-core/src/main/java/io/sentry/core/Hub.java

```diff
@@ -1,13 +1,34 @@
 package io.sentry.core;
 
 import io.sentry.core.protocol.SentryId;
+import java.util.Deque;
+import java.util.concurrent.ConcurrentLinkedDeque;
 
 public class Hub implements IHub {
+
+  private class StackItem {
+    public ISentryClient client;
```

Comment:
If this is supposed to be used from multiple threads and is mutable, then it should be `volatile`. Since this class is internal, `public` doesn't make sense and it can be left either package private or even `private`.